source/Information.h

```diff
@@ -14,11 +14,13 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #define INFORMATION_H_
 
 #include "Color.h"
+#include "Font.h"
```

Comment:
Would it make sense to break out the `Layout` class from `Font`? Doing so would mean we have a "lightweight" include for when all a class's interface needs to know is how we represent a layout, rather than how we draw fonts. The penalty is then that we need to have more explicit includes in more files, but at the same time, we only need to include the bits that are used.

If you choose to do that, I'm fine with grouping the font-related helper classes in a `fonts` subfolder. (If you also move the Font.cpp & Font.h files in this PR and they are shown as "deleted & added" instead of renamed, I would  recommend waiting for a follow-up PR to move those 2 files.)

Reply:
I plan to create ```DisplayText.h``` as a lightweight include.